tests/gateway_with_yggdrasil_ip/gatway_with_yggip_test.go

```diff
@@ -1,10 +1,7 @@
-//go:build integration
```

Comment:
integration tag is apparently needed, since it is used when building in the terratest workflow